README.md

```diff
@@ -1,7 +1,12 @@
 <p align="center">
-  <a href="https://fingerprintjs.com">
-    <img src="https://raw.githubusercontent.com/fingerprintjs/fingerprintjs-pro-svelte/main/resources/logo.svg" alt="FingerprintJS" width="312px" />
+  <a href="https://fingerprint.com">
+    <picture>
+     <source media="(prefers-color-scheme: dark)" srcset="resources/logo_light.svg" />
+     <source media="(prefers-color-scheme: light)" srcset="resources/logo_dark.svg" />
```

Comment:
Why didn't you use an absolute path here?

Reply:
It seems that the <source> element doesn't work with absolute paths, at least not on GitHub

Reply:
Sounds strange, because it is a standard HTML tag with a [standard property](https://www.w3schools.com/tags/att_source_srcset.asp)